### PR TITLE
Feature - Delete primary scenario

### PR DESCRIPTION
**Pull request by khaitruong922**

### CODE REVIEW

**Review by FabienLelaquais — APPROVED** [review]
Looks neat.

**Review by jrobinAV — APPROVED** [review]
Just perfect!

**Comment by jrobinAV** [comment]
By the way, it s a good improvement to share. can you add an entry in the release notes (relnotes.md file) for Core 1.2 community edition?

File: docs/relnotes.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ This is the list of changes to Taipy releases as they were published.
    - The data node of a scenario or a pipeline can now be accessed at the scenario or pipeline levels.
    - When submitting a scenario, a pipeline or a task, a list of jobs or a Job will be returned.
    - A submitted job can be canceled. When canceling the job, its subsequent jobs will be abandoned. When the cancel method is called on a running job, it cannot be canceled, but its subsequent jobs will be abandoned.
+   - A primary scenario can deleted along with its cycle if it is the only scenario in the cycle.
```
Comment (jrobinAV) [inline docs/relnotes.md]:
```suggestion
   - A primary scenario can be deleted along with its cycle if it is the only scenario in the cycle.
```

**Review by jrobinAV — APPROVED** [review]

**Pull request merged**